https://issues.redhat.com/browse/ACM-15075 (#7180)
open-cluster-management/rhacm-docs: release_notes/known_issues_global.adoc

There is currently no workaround for this issue. 

> [#search-not-enabled]

Author: Dev reviewers:

Looking at the original PR, I shortened the anchor to make it a bit easier to find. And typically, we want the latest known issues at the top of the doc, instead of the bottom. Let me know what you think :)

== The {global-hub} search is not enabled 
//2.12:15075

> If you installed your {mce-short} into a customized namespace, then you cannot enable the {global-hub} search, because in this case, the search operator failed to add the `globalSearchFeatureFlag`. 

Author: Dev reviewers:

Please let em know if I am correct that this is a result of the user action -- if the user installed the mce short into the customized namespace, and then, the user cannot enable the global hub serach.

Reviewer: yes, the search operator will failed to update the console-mce-config configmap by adding the `globalSearchFeatureFlag=true`.

> There is currently no workaround for this issue. 

Author: Correct me if there are in fact workaround steps we can provide users, thanks!

Reviewer 2: Workaround is to manually patch the console-mce-config ConfigMap in the correct namespace.

```
oc patch configmap console-mce-config -n {{MCE_NAMESPACE}} -p '{"data": {"globalSearchFeatureFlag": "enabled"}}'
```

There is currently no workaround for this issue. 

[#search-not-enabled]

> == The {global-hub} search is not enabled 

Reviewer 3: ```suggestion
== The {global-hub} Search is not enabled 
```

//2.12:15075

> If you installed your {mce-short} into a customized namespace, then you cannot enable the {global-hub} search. In this case, the search operator fails to update the `console-mce-config` config map because it added the `globalSearchFeatureFlag=true`, preventing you from enabling the {global-hub} search.

Reviewer 3: ```suggestion
If you installed your {mce-short} into a customized namespace, then you cannot enable the {global-hub} Search. In this case, the search operator fails to update the `console-mce-config` config map because the `globalSearchFeatureFlag=true` flag is added, preventing you from enabling the {global-hub} Search.
```

To workaround this issue, you must manually place the `console-mce-config` config map in the correct namespace by running the following command: 

> ----
> oc patch configmap console-mce-config -n {{MCE_NAMESPACE}} -p '{"data": {"globalSearchFeatureFlag": "enabled"}}'
> ----

Reviewer 3: ```suggestion
[source,bash]
----
oc patch configmap console-mce-config -n {{MCE_NAMESPACE}} -p '{"data": {"globalSearchFeatureFlag": "enabled"}}'
----
```
